### PR TITLE
Corrected the asserted properties of space S000107

### PR DESCRIPTION
**Pull request by gb-solis**
Follow-up to the discussion in issue https://github.com/pi-base/data/issues/117.
N.b. that the citation style used in the "name" field of property P000008 might need revision.

Also fixed a minor grammar mistake I spotted in another space.

@StevenClontz, @prabau, @jamesdabbs

### CODE REVIEW

File: spaces/S000107/properties/P000008.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 uid: T001134
 space: S000107
 property: P000008
-value: true
+value: false
 refs:
-- doi: 10.1007/978-1-4612-6290-9_6
-  name: Counterexamples in Topology
+- doi: 10.4064/fm-88-2-127-132
+  name: VAN DOWEN, Eric; The box product of countably many metrizable spaces need not be normal
```
Comment (prabau) [inline spaces/S000107/properties/P000008.md:8]: Typo, and would prefer not all caps.  It's actually "van Douwen, Eric" (with a lowercase "v", it's Dutch.  See the pdf)

Comment (gb-solis) [reply spaces/S000107/properties/P000008.md]: Done

**Comment by gb-solis** [comment]
Apart from prabau's comments (thanks!), I haven't heard back from you. Is there something else I should be doing?

**Review by StevenClontz — APPROVED** [review]

**Comment by StevenClontz** [comment]
Nope, I was just slow to review this. Thanks everyone for the correction.

**Comment by prabau** [comment]
Thanks Steven.  That also seems to have unblocked the previous two pull requests, that had previous successful builds, but did not show up on https://topology.pi-base.org for some reason, but now they all do.  Back to normal!
@jamesdabbs @ccaruvana

**Pull request merged**